lib/hotel_block.rb

```diff
@@ -0,0 +1,64 @@
+module Hotel
+  class HotelBlock
+    attr_reader :room, :check_in, :check_out, :cost, :reservations, :hotel
```

Comment:
Looks like you got started here, but didn't get far.  

Just one observation, I think the `hotel` instance method is unneeded.  You should have something to keep track of the rooms in the block however.